components/SSI/DIDxWallet/Wallet/Balances/index.tsx

```diff
@@ -1216,7 +1244,10 @@ function Component() {
                                                 <div
                                                     className={styles.btnAction}
                                                     onClick={() =>
-                                                        withdrawFunds('ZIL')
+                                                        withdrawFunds(
```

Comment:
Hi @devwistkey, how can we avoid having the withdrawFunds duplicated? Is it due desktop vs mobile or what is the reason?

Reply:
Hi @tralkan , yes you're right. That's because desktop and mobile version